app/models/usuario.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
+    {
+      name: { type: DataTypes.STRING, allowNull: false },
+      last_name: { type: DataTypes.STRING, allowNull: false },
```

Comment:
We usually use camelCase in the internal code, so the model should respect that.
UYou can use `field` option to associate the model attribute to the table attribute, for example: `lastName: { type: DataTypes.STRING, allowNull: false, field: 'last_name' }`